docs/targets.md

```diff
@@ -5,6 +5,6 @@ Available targets:
   help                                Help screen
   help/all                            Display help for all targets
   help/short                          This help short screen
-  lint                                Lint terraform code
+  lint:
                              Lint terraform code

```

Comment:
@bnippercg Can you remove this change while fixing your merge conflicts too?

Reply:
Sure thing. I'll update that with the next push.